src/peergos/server/mutable/HttpMutablePointerServer.java

```diff
@@ -89,7 +89,7 @@ void setPointer(DataInputStream din, DataOutputStream dout) throws Exception
 
         void getPointer(DataInputStream din, DataOutputStream dout) throws Exception
         {
-            PublicKeyHash encodedSharingKey = PublicKeyHash.fromCbor(CborObject.deserialize(new CborDecoder(din)));
+            PublicKeyHash encodedSharingKey = PublicKeyHash.fromCbor(CborObject.deserialize(new CborDecoder(din), 1024));
```

Comment:
Ah, good of ``2^10``. How  about a static constant?

Reply:
done